pkg/inject/virtualgateway_envoy_test.go

```diff
@@ -818,6 +819,94 @@ func Test_virtualGatewayEnvoyMutator_mutate(t *testing.T) {
 				},
 			},
 		},
+		{
```

Comment:
We're also adding support for `DataDog` and `StatsD` for VG's envoy in this diff, so let's add test cases for those as well.

Reply:
Okay, will add test cases as well

Reply:
You need to split this up. Multiple tracer configs aren't supported. https://github.com/aws/aws-app-mesh-controller-for-k8s/blob/a3d31b456f6e99b526b0f3ecfe9c27d289767bfb/pkg/inject/config.go#L170

Reply:
Fixing it